doc/source/user_guide/ug_implementation.rst

```diff
@@ -1156,8 +1156,11 @@ io package.  The namelist variable ``history_format`` further refines the
 format approach or style for some io packages.
 
 Model output data can be written as instantaneous or average data as specified
-by the ``hist_avg`` namelist array and is customizable by stream.  The data is 
-written at the period(s) given by ``histfreq`` and
+by the ``hist_avg`` namelist array and is customizable by stream. Characters
+can be added to the "h" character to distinguish the streams. This can be changed
+by modifying ``hist_str`` to something other than "x".
```

Comment:
I think it might be more appropriate to introduce `hist_str` a little below, after we introduce `history_file`, and we explain the different frequencies.

I think the best location would be after the sentence "The output period can be
discerned from the filenames.", at lines 1202-1203:

https://github.com/CICE-Consortium/CICE/blob/d05b1f30e57d2531799d4dbfb53464422582dd6f/doc/source/user_guide/ug_implementation.rst?plain=1#L1202-L1203